device-types/DataDirect Networks/SFA7990-FC16.yaml

```diff
@@ -0,0 +1,81 @@
+---
+manufacturer: DataDirect Networks
+model: SFA7990-FC16
+slug: sfa7990-fc16
```

Comment:
All slugs should have the manufacturer appended. In this case, please append `datadirect-networks-`